src/tersets.zig

```diff
@@ -92,21 +91,23 @@ pub fn decompress(
 
     const method_index: u8 = compressed_values[compressed_values.len - 1];
 
```

Comment:
Maybe move this empty line since Line 93 and Line 94 are very related.

Reply:
Fixed.